package.json

```diff
@@ -64,7 +64,7 @@
     "vue-cli-plugin-i18n": "~2"
   },
   "engines": {
-    "node": "18",
-    "npm": "9"
+    "node": "20",
+    "npm": "10"
```

Comment:
I'd expect to see these change in package-lock.json as well.

Reply:
Thank you!

Two observations:
- It's hard for me to remember to install dependencies in frontend when switching from backend where the `make` command takes care of it
- I'm in the habit of using `npm ci` but of course that doesn't update engines